global_costmap:
   global_frame: /map
   robot_base_frame: /base_footprint
   update_frequency: 1.0
   publish_frequency: 0.5
   static_map: false
   rolling_window: true
   transform_tolerance: 0.5

   plugins:
    -
      name: footprint_layer
      type: "costmap_2d::FootprintLayer"
    -
      name: obstacle_layer
      type: "costmap_2d::ObstacleLayer"
    -
      name: inflation_layer
      type: "costmap_2d::InflationLayer"
    -
      name: ball_picker_layer
      type: "costmap_2d::BallPickerLayer"

